library/modules/bigip_gtm_wide_ip.py

```diff
@@ -409,6 +484,50 @@ def aliases(self):
         self._values['aliases'].sort()
         return self._values['aliases']
 
+    @property
+    def persistence(self):
```

Comment:
This is incorrect, this is how you should do the boolean:

https://github.com/F5Networks/f5-ansible/blob/devel/library/modules/bigip_profile_http.py#L1035

Reply:
Oh that's an interesting function for tackling boolean! I'll use it!